exercises/concept/high-school-sweetheart/.docs/instructions.md

```diff
@@ -46,26 +35,15 @@ HighSchoolSweetheart.initials("Lance Green")
 # => "L. G."
 ```
 
-## 4. Put the initials inside of the heart
+## 4. Put the initials inside of the ASCII box
 
-Implement the `HighSchoolSweetheart.pair/2` function. It should take two full names and return the initials inside an ASCII heart. Make sure to reuse `HighSchoolSweetheart.initials/1` that you defined in the previous step.
+Implement the `HighSchoolSweetheart.pair/2` function. It should take two full names and return the initials inside an ASCII box. You don't need to draw the box yourself - make use of the box included in the code comment. Make sure to reuse `HighSchoolSweetheart.initials/1` that you defined in the previous step.
 
 ```elixir
 HighSchoolSweetheart.pair("Blake Miller", "Riley Lewis")
 # => """
-#      ******       ******
-#    **      **   **      **
-#  **         ** **         **
-# **            *            **
-# **                         **
-# **     B. M.  +  R. L.     **
-#  **                       **
-#    **                   **
-#      **               **
-#        **           **
-#          **       **
-#            **   **
-#              ***
-#               *
+# ❤-------------------❤
+# |  B. M.  +  R. L.  |
+# ❤-------------------❤
```

Comment:
I wanted to include it without any spaces:
```suggestion
#❤-------------------❤
#|  B. M.  +  R. L.  |
#❤-------------------❤
```

But the Elixir formatter adds the spaces